### ISSUE TITLE
Angular 13 - Component factory argument

### ISSUE THREAD

**Issue by romankh** [lentschi/ngx-nested-ellipsis#6]
Hi, 

after upgrade to Angular 13 I noticed an error that occurs only in development profile (for now).
It seems like the component factory is called with a deprecated argument combination.

In production profile it still builds and works, but I guess it can be expected to break with upcoming releases.

Error message is as following:

```
ERROR Error: ASSERTION ERROR: It looks like Component factory was provided as the first argument and an options object as the second argument. This combination of arguments is incompatible. You can either change the first argument to provide Component type or change the second argument to be a number (representing an index at which to insert the new component's host view into this container) [Expected=> true == false <=Actual]
    at throwError (core.mjs:326)
    at assertEqual (core.mjs:277)
    at ViewContainerRef.createComponent (core.mjs:23151)
    at NestedEllipsisDirective.restoreView (ngx-nested-ellipsis.js:217)
    at NestedEllipsisDirective.ngOnInit (ngx-nested-ellipsis.js:137)
```

**Comment by lentschi** [comment]
@romankh Thanks for pointing that out!

Fixed in `ngx-nested-ellipsis@2.0.0`

**Comment by romankh** [comment]
@lentschi Thanks for the fix, works like a charm
